tests/test_lochness.py

```diff
@@ -135,9 +135,14 @@ def config_load_test(f: 'location', archive_base=None):
 
     if archive_base:
         Lochness['phoenix_root'] = archive_base
+
     if 'phoenix_root' not in Lochness:
         raise config.ConfigError('need either --archive-base or '
                           '\'phoenix_root\' in config file')
+
+    if 'BIDS' not in Lochness:
+        Lochness['BIDS'] = False
```

Comment:
Didn't we set it to `True` somewhere in the above?

Reply:
Yes, we set the `BIDS = True` in the `lochness_create_template.py`. Here I'm adding `Lochness['BIDS'] = False` for this function to be used with other configuration settings where `BIDS` is not included in the `config.yml`